lib/checks/language/xml-lang-mismatch.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "xml-lang-mismatch",
```

Comment:
Should we call this html-xml-lang-mismatch, to distinguish it from the one where we look at all other elements that have lang and xml:lang?

Reply:
Not right now. When we refactor other lang rules, can do that as we will have better context of all rules involved. I will leave the rule name and file names as is, in this PR.

I have created an issue for this - https://github.com/dequelabs/axe-core/issues/1066

Reply:
Sorted!